core/admin/admin-notices.php

```diff
@@ -544,6 +559,17 @@ public function admin_notices() {
 				return;
 			}
 		}
+
+		/** @var Base_Notice $notice_instance */
+		foreach ( $this->get_class_notices() as $notice_instance ) {
+			if ( ! $notice_instance->should_print() ) {
+				continue;
+			}
+
+			$this->print_admin_notice( $notice_instance->get_config() );
```

Comment:
```suggestion
			$notice_instance->print();
```
form the `print` check if `should_print`.